### PR TITLE
Feature/layout changes

### PR DESCRIPTION
**Pull request by wolf31o2**
When using [omnibus](https://github.com/opscode/omnibus) to build packages, some base libraries between provisioner and ui conflict, namely openssl and ncurses. This moves things from `bin` and `distribution` under the appropriate project subdirectory. The `loom-service` init script template is now a proper script under each project.

### CODE REVIEW

File: provisioner/distribution/etc/init.d/loom-provisioner [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[ -f /etc/default/loom-provisioner ] && source /etc/default/loom-provisioner
+
+# create pid dir owned by loom user
+mkdir -p /var/run/loom
+chown -R loom:loom /var/run/loom
+
+# drop permissions to loom user and run service script
+
+if [[ $UID -eq 0 ]]; then
+    su loom -c "cd /opt/loom/provisioner/bin; exec ./loom-provisioner.sh $*"
```
Comment (albertshau) [inline provisioner/distribution/etc/init.d/loom-provisioner]: didn't the script get renamed to provisioner.sh? or is that different thing

File: server/distribution/etc/init.d/loom-server [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+[ -f /etc/default/loom-server ] && source /etc/default/loom-server
+
+# create pid dir owned by loom user
+mkdir -p /var/run/loom
+chown -R loom:loom /var/run/loom
+
+# drop permissions to loom user and run service script
+
+if [[ $UID -eq 0 ]]; then
+    su loom -c "cd /opt/loom/server/bin; exec ./loom-server.sh $*"
```
Comment (albertshau) [inline server/distribution/etc/init.d/loom-server]: similar question about loom-server.sh vs server.sh

**Comment by albertshau** [comment]
few questions, otherwise :+1:

**Comment by wolf31o2** [comment]
Fixed in a subsequent commit.

**Pull request merged**